package.json

```diff
@@ -37,6 +37,6 @@
   "type": "module",
   "optionalDependencies": {},
   "engines": {
-    "node": ">=14.0.0"
+    "node": ">=20.0.0"
```

Comment:
node20 isn't LTS yet (or wait 8 more days)

Reply:
yeah it's close enough I think it's fine to just go ahead with it